assets/js/admin/quick-edit.js

```diff
@@ -57,7 +57,7 @@ jQuery(
 					'select[name="_visibility"] option, ' +
 					'select[name="_stock_status"] option, ' +
 					'select[name="_backorders"] option'
-				).prop( 'selected', false );
+				).removeAttr( 'selected' );
```

Comment:
```suggestion
				).prop( 'selected', false ).removeAttr( 'selected' );
```

Reply:
Or how about `)[0].removeAttribute( 'selected' );` The `[0]` gets the actual DOM element and from there we could use the vanilla JS function of `removeAttribute()`.

Reply:
I don't think that'll work in this case, as the selector has the potential to match multiple options, so you'd have to iterate over the set of results... but not only that, I tested that on a smaller scale, and it doesn't appear to actually deselect the option.

I think it's fine as you originally had it. Behind the scenes, `$( selector ).removeAttr` appears to function differently than the native `removeAttribute()`.

Reply:
So I actually had this:

```
				$(
					'select[name="_tax_status"] option, ' +
					'select[name="_tax_class"] option, ' +
					'select[name="_visibility"] option, ' +
					'select[name="_stock_status"] option, ' +
					'select[name="_backorders"] option'
				).each( function() {
					$( this )[0].removeAttribute( 'selected' );
				} );
```

Reply:
But I think we could go with your suggestion as it is cleaner.

Reply:
Ahh, yep, that appears to work. 👍